### DISCUSSION TITLE
Agent version in admin panel

### DISCUSSION THREAD

**Discussion by zc-devs** [woodpecker-ci/woodpecker#2237] [Request a Feature] [answered]
Maybe it's good idea to display agent version in admin settings.

Something like this:
![Screenshot 2023-08-17 151345](https://github.com/woodpecker-ci/woodpecker/assets/127358482/52d8bd4c-56dd-411e-98a0-80d69ed6b5fe)

**Comment by qwerty287** [comment, marked as answer]
@zc-devs You can view the agent version if you click the "edit" button (there are a lot more details even though you can't edit all of them)
